chapters/Chapter 2 — Dressara.md

```diff
@@ -0,0 +1,26 @@
+The women of Dressara often wore dresses, which Leanne both admired and found a shame. Leanne enjoyed wearing dresses, but found women in suits much more handsome.
```

Comment:
I'd like to start this chapter with something more like action. Something more present, rather than descriptive.

I'm not sure if that means I want a different first sentence, or if I want to add stuff after this sentence, mingled in with the next paragraph.

Also, the name of this other person who Leanne has tracked down? I don't know it. It's important, because it is my secondary character! Ugh.